### PR TITLE
[Doc Feature][Change notification for recording available - ODSP][3099832]

### PR DESCRIPTION
**Pull request by v-sdhakshina**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/424d10c4ca5d6306898f3a34eda175c65f709326/msteams-platform%2FTOC.yml) | 99 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3498aee3-114d-4d00-9744-31e46b05c0b4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/meeting-transcripts/fetch-id.md](https://github.com/MicrosoftDocs/msteams-docs/blob/424d10c4ca5d6306898f3a34eda175c65f709326/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Ffetch-id.md) | 94 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8bc7e56e-c043-4d2b-9988-9d5c3fa84b4d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/424d10c4ca5d6306898f3a34eda175c65f709326/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 100 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5768c186-1e03-491e-87f1-81e3ceb621e9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md [hunk 1]
```diff
@@ -1,54 +1,56 @@
...
 > [!NOTE]
 > In the future, Microsoft may require you or your customers to pay additional fees based on the amount of data accessed through the API.
 
-To fetch the transcript for a particular meeting:
+To fetch the transcript and recording for a particular meeting:
 
-- [Configure permissions on Azure AD to access transcript](#configure-permissions-on-azure-ad-to-access-transcript)
+- [Configure permissions on Azure AD to access transcript and recording](#configure-permissions-on-azure-ad-to-access-transcript-and-recording)
 - [Obtain meeting ID and organizer ID](fetch-id.md)
+- [Get notified when transcripts is available](/graph/transcripts)
```
Comment (Anjali-Patle) [inline msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md:27]: Should this be changed to 'Get notified when transcripts and recordings are available'?

File: msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md [hunk 2]
```diff
@@ -1,54 +1,56 @@
...
 - [Use Graph APIs to fetch transcript](/graph/api/resources/calltranscript)
```
Comment (Anjali-Patle) [inline msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md:28]: Same as above, should recording be added here?

File: msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md [hunk 3]
```diff
@@ -1,54 +1,56 @@
...
 
-After you've configured the permissions, configure your app to receive change notifications for all relevant meeting events. Notifications contain meeting ID and organizer ID that help in accessing transcript content. Your app can fetch the transcript for a meeting when it's generated after it ends. The content of the transcript is available as `.vtt` or `.docx` file.
-
-> [!NOTE]
-> The docx format for transcripts will be deprecated starting May 31, 2023. For more information, see [Get callTranscript](/graph/api/calltranscript-get).
+After you've configured the permissions, configure your app to receive [change notifications](/graph/transcripts) for [transcripts available events](/graph/transcripts) or all relevant meeting events. Notifications contain meeting ID and organizer ID that help in accessing transcript content and recording. Your app can fetch the transcript and recording for a meeting when it's generated after it ends. The content of the transcript is available as `.vtt` file. The recording of the meeting is available as an `.mp4` file.
```
Comment (Anjali-Patle) [inline msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md:46]: Please change this to 'After you've configured the permissions, configure your app to receive [change notifications](https://review.learn.microsoft.com/en-us/graph/transcripts-and-recordings.md) for[ transcripts and recordings available events](https://review.learn.microsoft.com/en-us/graph/transcripts-and-recordings) or all relevant meeting events.'

File: msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md [hunk 4]
```diff
@@ -1,54 +1,56 @@
...
 
 For more information about how your app can know when the meetings ends, see [subscribe to change notifications](fetch-id.md#subscribe-to-change-notifications) and [use Bot Framework to get meeting ID and organizer ID](fetch-id.md#use-bot-framework-to-get-meeting-id-and-organizer-id).
 
+For more information about how your app can know when a transcript or recording is available after the online meeting ends, see [get notified when transcripts is available](/graph/transcripts).
```
Comment (Anjali-Patle) [inline msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md:50]: Please change to 'get notified when transcripts and recordings are available'

**Review by Anjali-Patle — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/af3b88dfef1bd74781ccb8235eed9c0dfc0390b6/msteams-platform%2FTOC.yml) | 99 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/35dee078-4187-47d3-99e6-c79784e782f2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/meeting-transcripts/fetch-id.md](https://github.com/MicrosoftDocs/msteams-docs/blob/af3b88dfef1bd74781ccb8235eed9c0dfc0390b6/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Ffetch-id.md) | 94 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3c24cdab-d7ec-4b52-889a-f965cf1aeb34) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/af3b88dfef1bd74781ccb8235eed9c0dfc0390b6/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 100 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c991c1eb-711e-403a-9177-2ae44eecd80e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md (hunk 1 above)
Comment (v-sdhakshina) [reply msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md:27]: updated as suggested

File: msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md (hunk 2 above)
Comment (v-sdhakshina) [reply msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md:28]: updated as suggested

File: msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md (hunk 3 above)
Comment (v-sdhakshina) [reply msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md:46]: updated as suggested

File: msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md (hunk 4 above)
Comment (v-sdhakshina) [reply msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md]: Updated as suggested

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/36e4a5e5e68ff94340d0f104cc5b3fff4e54d94a/msteams-platform%2FTOC.yml) | 99 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4c6e2c14-5cbf-4d1c-bfb0-73fe4459d590) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/meeting-transcripts/fetch-id.md](https://github.com/MicrosoftDocs/msteams-docs/blob/36e4a5e5e68ff94340d0f104cc5b3fff4e54d94a/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Ffetch-id.md) | 94 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c0595859-7bfe-4812-b1b2-1534f0c6a883) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/36e4a5e5e68ff94340d0f104cc5b3fff4e54d94a/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 100 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/119aa188-c129-4948-8195-326d07c8eeb1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/c877513bce21b3b7435f93c10cb767278de56e99/msteams-platform%2FTOC.yml) | 99 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/aaff3263-8809-4501-bbf6-32118ccd9ca3) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/meeting-transcripts/fetch-id.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c877513bce21b3b7435f93c10cb767278de56e99/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Ffetch-id.md) | 94 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a0f5521f-ef02-4369-9cd0-9eecbdcfff77) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c877513bce21b3b7435f93c10cb767278de56e99/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 100 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c99c02f0-ebc7-4a26-a8f9-9f405c985d92) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Anjali-Patle — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/4f4a31ef3df4f081eba58789fd4f45d43cfa1251/msteams-platform%2FTOC.yml) | 99 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f89cf7bc-f9bb-4f5a-8361-6ad37809dfc4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/meeting-transcripts/fetch-id.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4f4a31ef3df4f081eba58789fd4f45d43cfa1251/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Ffetch-id.md) | 94 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d8edd2d4-edf0-493c-92cf-9116b41d2634) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4f4a31ef3df4f081eba58789fd4f45d43cfa1251/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 100 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a32ea62c-c622-4612-9ee9-d58a5baa7b14) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/af64ad0c5a218f28a63313ab534eed1fa974b50b/msteams-platform%2FTOC.yml) | 99 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7a3a9524-4fad-47dd-ae2d-00e1a0f5db9b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/af64ad0c5a218f28a63313ab534eed1fa974b50b/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 100 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/de8b2be2-a1a7-4d6b-a1dc-8f8abc6a887c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/521e3c37725281062aa8b356cc29fb2f760994fe/msteams-platform%2FTOC.yml) | 99 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e8e4e425-7138-49b6-9727-5010023e5752) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/521e3c37725281062aa8b356cc29fb2f760994fe/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 100 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/143e5b71-9a20-47e4-b7ef-ad802ad14eca) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/ed9bfacebcde7c24f46cba6ed337da91818f8e81/msteams-platform%2FTOC.yml) | 99 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/37964313-820b-4185-8579-6b07b8072762) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ed9bfacebcde7c24f46cba6ed337da91818f8e81/msteams-platform%2Fgraph-api%2Fmeeting-transcripts%2Foverview-transcripts.md) | 100 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6f52fdbf-0f15-4208-8bb0-d8aedb408757) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**